network/data-indexers/README.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Ecosystem Data Indexer Providers
+sidebar_position: 70
+---
+
+To power your applications, a range of data services will be available from data indexers with query interfaces. These indexers will allow you to easily store and retrieve decoded ledger data, contract event emissions, and more. You can focus more energy on making your project successful, and waste less time figuring out precisely how to get the important data from the network.
+
+### [SubQuery](https://subquery.network/)
+
+An open-source data indexer that supports both Stellar and Soroban. SubQuery allows users to build a custom API for their dapp or smart contract and provides clean indexed data to any service. You can index ledgers, transactions, operations, and effects from Stellar and transactions and events from Soroban into your own Postgres database. Read the quick start guide here.
+
+### [Mercury](https://mercurydata.app/)
+
+A network indexing service that provides a toolkit to help get you started working with Stellar network data.
+
+### [BlockEden](https://blockeden.xyz/stellar-soroban/)
+
+An all-in-one Soroban RPC, indexer GraphQL API, and data analytics web portal.
+
+### [Goldsky](https://goldsky.com/)
+
+Mirror and indexer coming soon.
+
+### [Ankr](https://www.ankr.com/)
+
+Also offers RPC and Horizon instances.
+
+### [Zettablock](https://www.zettablock.com/)
```

Comment:
Is Zettablock actually available for Stellar? I see that we're included in the little scrolling banner area on their site, but in the docs, dashboard, blog, anywhere else on their site I can't seem to find any way to use Stellar data.

I see we added this link a month ago, though, so I'm not sure if something has changed since then.

Reply:
If we do want to keep this here, we may want to figure out an accurate description for it